__tests__/lib/__snapshots__/commands.test.ts.snap

```diff
@@ -29,10 +29,15 @@ Object {
         "name": "openapi",
         "position": 1,
       },
+      Object {
+        "description": "Reduce an OpenAPI definition into a smaller subset.",
+        "name": "openapi:reduce",
+        "position": 3,
+      },
       Object {
         "description": "Alias for \`rdme openapi\`. [deprecated]",
         "name": "swagger",
-        "position": 2,
```

Comment:
I'm surprised that this has always been showing up after `validate`  in `--help` with them having the same position.

Reply:
:lots-of-yikes: